tools/debian/convert_debian.py

```diff
@@ -34,97 +36,179 @@
 # e.g. [buster] - xz-utils 5.2.4-1+deb10u1
 VERSION_PATTERN = re.compile(r'\[(.*?)\]\s*-\s*([^\s]+)\s*([^\s]+)')
 
+# e.g. <define-tag moreinfo>\n Some html here \n</define-tag>
+WML_DESCRIPTION_PATTERN = re.compile(r'<define-tag moreinfo>((?:.|\n)*)</define-tag>', re.MULTILINE)
 
-class VersionInfo:
-    """Debian version info."""
+# e.g.
+WML_REPORT_DATE_PATTERN = re.compile(r'<define-tag report_date>(.*)</define-tag>')
+
+
+class DebianSpecificInfo:
     release: str
+
+    def __init__(self, release: str):
+        self.release = release
+
+    def __repr__(self):
+        return json.dumps(self, default=dumper)
+
+
+class AffectedInfo:
+    """Debian version info."""
+    ecosystem_specific: DebianSpecificInfo
     package: str
+    ranges: [str]
     fixed: str
+    versions: [str]
 
     def __init__(self, release: str, package: str, fixed: str):
-        self.release = release
+        self.ecosystem_specific = DebianSpecificInfo(release)
         self.package = package
         self.fixed = fixed
 
+    def toJSON(self):
```

Comment:
please use snake_case for all function names.

Also, can we just use `__dict__` for this? That's more accurately describes what this is doing and makes the `dumper` below much simpler.

Reply:
This is resolved.